app/controllers/actions/AuthAction.scala

```diff
@@ -19,7 +19,7 @@ package controllers.actions
 import com.google.inject.{ImplementedBy, Inject}
 import play.api.mvc.Results._
 import play.api.mvc.{ActionBuilder, ActionFilter, AnyContent, BodyParser, ControllerComponents, Request, Result}
-import uk.gov.hmrc.auth.core.retrieve.Retrievals._
+import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals._
```

Comment:
Looks interesting - whats this library for?

Reply:
This is Auth. They deprecated the old Retrievals and moved to the `v2` package. This has been done for all our services some time ago.